protos/chaincode.proto

```diff
@@ -70,6 +70,7 @@ message ChaincodeSpec {
     string secureContext = 5;
     ConfidentialityLevel confidentialityLevel = 6;
     bytes metadata = 7;
+    map<string, string> attributes = 8;
```

Comment:
Should the attributes be part of the fabric.Transaction message instead of a ChaincodeSpec?